support-frontend/stories/checkoutLayout/CheckoutBenefitsList.stories.tsx

```diff
@@ -7,54 +7,54 @@ import { checkListData } from 'components/checkoutBenefits/checkoutBenefitsListD
 import { Box, BoxContents } from 'components/checkoutBox/checkoutBox';
 
 export default {
-	title: 'Checkouts/Benefits List',
-	component: CheckoutBenefitsList,
-	argTypes: {
-		handleButtonClick: { action: 'button clicked' },
-	},
-	decorators: [
-		(Story: React.FC): JSX.Element => (
-			<Container backgroundColor={neutral[97]}>
-				<Columns
-					collapseUntil="tablet"
-					cssOverrides={css`
+  title: 'Checkouts/Benefits List',
+  component: CheckoutBenefitsList,
+  argTypes: {
+    handleButtonClick: { action: 'button clicked' },
```

Comment:
It looks like the tab characters in this file have been replaced with double spaces, it would be good to undo this. If you get Prettier and ESLint set up properly in your editor that should be able to auto-fix it for you.